src/libraries/System.Diagnostics.EventLog/tests/Helpers.cs

```diff
@@ -13,8 +14,9 @@ namespace System.Diagnostics.Tests
 {
     internal class Helpers
     {
-        public static bool NotElevatedAndSupportsEventLogs { get => !AdminHelpers.IsProcessElevated() && SupportsEventLogs; }
-        public static bool IsElevatedAndSupportsEventLogs { get => AdminHelpers.IsProcessElevated() && SupportsEventLogs; }
+        public static bool HasAssemblyFilesIsElevatedAndSupportsEventLogs => PlatformDetection.HasAssemblyFiles && IsElevatedAndSupportsEventLogs;
+        public static bool NotElevatedAndSupportsEventLogs { get => !RemoteExecutor.IsSupported && SupportsEventLogs; }
+        public static bool IsElevatedAndSupportsEventLogs { get => RemoteExecutor.IsSupported && SupportsEventLogs; }
```

Comment:
I have to revert these, they are used by lots of tests that do not use remote executor.

```suggestion
        public static bool NotElevatedAndSupportsEventLogs { get => !AdminHelpers.IsProcessElevated() && SupportsEventLogs; }
        public static bool IsElevatedAndSupportsEventLogs { get => AdminHelpers.IsProcessElevated() && SupportsEventLogs; }
```